tests/console/yast2_nfs4_client.pm

```diff
@@ -51,6 +51,11 @@ sub run {
     assert_script_run 'sed -i \'5i# test comment\' /etc/fstab';
     assert_script_run 'cat /etc/fstab > fstab_before';
 
+    if (is_tumbleweed) {
```

Comment:
Not great, but I don't know how to address better this issue

Reply:
is there any detection that we can do here? btw, do you have some VR?

Reply:
although if you refer to this: http://waaa-amazing.suse.cz/tests/10312#step/yast2_nfs4_client/40 mmm then it is tricky, perhaps a needle with `[  ` black background, white font, just guessing...

Reply:
Right, you acually found the failed run what I was searching :) ! I think it is too unpredictable. Though, this happened only with that build, I tried again with the latest build and it's no longer happening. maybe it's better to just remove it.